specifyweb/frontend/js_src/lib/components/FormEditor/Editor.tsx

```diff
@@ -176,6 +179,25 @@ export function FormEditorWrapper(): JSX.Element {
   );
 }
 
+function UseLabelsSchema(): JSX.Element {
```

Comment:
could you reuse this component with the Form Meta menu? or does the logic differ a bit?

Reply:
the logic differs so I had to create a new one.